onchain/plutarch/midgard-merkle.cabal

```diff
@@ -238,15 +227,20 @@ common dependencies
     , text
 
 executable midgard
-  import:         lang, dependencies
+  import:         lang
   main-is:        Main.hs
 
   -- Modules included in this executable, other than Main.
   -- other-modules:
 
   -- LANGUAGE extensions used by modules in this package.
   -- other-extensions:
-  build-depends:  midgard-merkle
+  build-depends:
+    , base
+    , midgard-merkle
```

Comment:
Where are you getting this dependency from? I don't see it in any of the `nix` files